coding-concepts.js

```diff
@@ -67,7 +67,7 @@ class Learn {
   }
 }
 var learnStudent = new Learn("George")
-// console.log(learnStudent)
+console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It will log {[student: George, cohort: Bravo, year: 2022]}
+// b) Verify and explain: I was wrong about it logging the key:value pairs in an array but I was correct concerning the output. In the pre-established class Learn, we have an object with three key:value pairs. Only one of them has been established as variable based on input, that one being this.student = name. Because name is also a parameter in the constructor function, we can pass in a name whenever we create a new class (learnStudent) and the output will reflect our input.
```

Comment:
Excellent work on this section!  Your explanations were spot on and well worded.